pbs/pbsrequest.go

```diff
@@ -81,6 +82,7 @@ type PBSAdUnit struct {
 	Params     json.RawMessage
 	Video      PBSVideo
 	MediaTypes []MediaType
+	Instl      int8
```

Comment:
Can we make this enum-like, for better compiler-safety? From your description in `pbs_request.json`, it looks like there are only two legal values.

See `MediaType` for an example.

Reply:
The open rtb library we use is taking int8 for Instl field, if we make it enum-like, then we need to parse int8 input to an enum then parse it back to int8 immediately, I think it's redundant.
I could potentially add a check that does not take in values other than 1 or 0, what do you think?

Reply:
The more I think about it... the more I think these changes are better in the openrtb library. They would benefit everyone who wants to send OpenRTB messages--Prebid included.

I [logged an issue](https://github.com/mxmCherry/openrtb/issues/15) about it... so let's see if they're amenable to the idea first. I don't think we need to hold this up waiting for them, though.